include/openvswitch/ovs-p4rt.h

```diff
@@ -15,6 +15,15 @@
 extern "C" {
 #endif
 
+/* Control OVS offload with an environment variable during runtime.
+ * If env variable OVS_P4_OFFLOAD=false, then disable OVS offload, else
+ * if OVS_P4_OFFLOAD is not set or OVS_P4_OFFLOAD is any value other
+ * than false, then by default enable OVS offload.
+ */
+#define ENABLE_OVS_P4_OFFLOAD                                                     \
```

Comment:
This is misleading. A bare #define is assumed to be a static value. This isn't a static value, it's a dynamic value.

My recommendation would be to define a static inline function in a different header file (maybe `p4ovs.h`?).

```c
static inline bool ovs_p4_offload_enabled(void) {
  const char* offload = getenv("OVS_P4_OFFLOAD");
  return (offload == NULL) || strcmp(offload, "false") != 0;
}
```

Reply:
Sure, can add an inline function too.